client/Wayland/wlfreerdp.c

```diff
@@ -146,6 +148,21 @@ static BOOL wl_pre_connect(freerdp* instance)
 	PubSub_SubscribeChannelDisconnected(instance->context->pubSub,
 	                                    (pChannelDisconnectedEventHandler) wlf_OnChannelDisconnectedEventHandler);
 
+	if (settings->Fullscreen)
+	{
+		// Use the resolution of the first display output
+		output = UwacDisplayGetOutput(context->display, 1);
```

Comment:
It doesn't error or anything, the resolution just comes back as 0px by 0px. Both of my test machines work great with 1 in that spot (one is Gnome and the other is Weston).